server/plugin/command.go

```diff
@@ -14,6 +14,7 @@ import (
 )
 
 const (
+	featureIssueCreation = "issue_creation"
```

Comment:
Let's use the plural here for consistency 
```suggestion
	featureIssueCreation = "issue_creations"
```

Reply:
Hello, the change has been made.